.github/actions/deploy-to-control-plane/action.yml

```
@@ -1,56 +1,61 @@
+        echo "sha_short=$(git rev-parse --short HEAD)" >> $GITHUB_OUTPUT
+        echo "Using commit SHA: $(git rev-parse --short HEAD)"
+
     - name: Deploy to Control Plane
+      id: deploy
       shell: bash
       run: |
-        echo "Deploying to Control Plane"
-        cpflow deploy-image -a ${{ inputs.app_name }} --run-release-phase --org ${{inputs.org}}  --verbose
+        # Create temp file for output
+        TEMP_OUTPUT=$(mktemp)
+        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)